code/__DEFINES/traits.dm

```diff
@@ -230,6 +230,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_BLOOD_CLANS "blood_clans"
 /// Species with this trait have markings (this SUCKS, remove this later in favor of bodypart overlays)
 #define TRAIT_HAS_MARKINGS "has_markings"
+/// Species with this trait use skin tones for coloration
+#define TRAIT_SKINTONES "skintones"
```

Comment:
```suggestion
#define TRAIT_USES_SKINTONES "skintones"
```

i think this is a bit more verbose? kinda fits in with the whole variable it was replacing. shouldn't be too hard to find and replace. if you think it's too nitty feel free to ignore but i think it's neat

Reply:
Eh, I like my variables more verbose. I'll do this.